SchoolNormalization.md

```diff
@@ -0,0 +1,235 @@
+SchoolNormalizationService
+=============
+
+Table of Contents
+_________
+- [Request Information](#request-information)
+- [Sample Response](#sample-response)
+- [Versioning](#versioning)
+- [Supported Countries](#supported-countries)
+
+
+
+#Request Information
+
+
+HTTP method: GET or POST
+Parameters (query/form):
+-        query (required) : The educational instutution information to be queried.
+-        country (optional) : 2 digit ISO country code of eductaionl institution. Accepted country codes are [here](#supported-countries)
+ 
+Example: https://api.careerbuilder.com/core/normalizedschools?query=georgia%20tech&country=us
+
+#Sample Response
+
+
+```
+{
+    "data": {
+        "normalized_schools": [
+            {
+                "normalized_school_name": "Georgia Institute of Technology",
+                "id": "53bff579e4b04710d09fa98d",
+                "country": "US",
+                "surface_form": "georgia tech",
```

Comment:
I just noticed this -- do we want to be exposing surface_form in our service? it's not in P&N, and afaik that's usually something that the data scientists use internally. you can mark it transient if you don't want it to serialize.